docs/manual/common/concepts/ServiceDiscovery.md

```diff
@@ -1,33 +1,51 @@
-# Service Discovery
+# Registering and discovering services
 
-Microservices must support location transparency: the ability to reside on multiple hosts for load balancing and resilience, to move from one host to another when failures occur, and to scale the number of hosts up and down as load changes. 
+To be resilient and scalable, a system must support location transparency.  This allows you to run instances of the same microservice on multiple hosts, to move instances from one host to another when failures occur, and to scale the number of hosts up and down as load changes. In such a reactive system where microservice instances keep moving, spawning and dying, both clients and other microservices need a way to locate available service instances. 
 
-If a microservice's instances keep moving, spawning and dying, a mechanism for the microservice client to locate an instance is required. A Service Registry maintains an up-to-date list of available microservices that includes the host and port for each instance of each microservice. As load changes, the system can spawn or destroy instances in any location and continue to satisfy requests because the Service Registry collaborates with the microservice's instances to maintain the lookup table updated.
+Lagom provides the following functionality for your microservice systems:
```

Comment:
I changed the bullet to say "during development" The subsections themselves make pretty clear that  people need to choose/implement  something for deployment.